enhance node status decision and filter

Omarabdul3ziz opened the pull request:
- edit the node report interval in filtering
- lowercase the key in node.power column while generating
- edit the decideNodeStatus methods
- edit the filter in tests to differentiate between down/standby nodes
- fix for https://github.com/threefoldtech/tfgrid-sdk-go/issues/232

Omarabdul3ziz committed:
enhance node status decision and filter

- edit the node report interval in filtering
- lowercase the key in node.power column while generating
- edit the decideNodeStatus methods
- edit the filter in tests to differentiate between down/standby nodes

Omarabdul3ziz committed:
remove unused code

alichaddad commented on grid-proxy/internal/explorer/converters.go:
```diff
@@ -11,16 +11,31 @@ import (
-	nodeUpInterval = -80 * time.Minute
+	// node report its state every 40 mins, if didn't report for 2 cycles, it's marked down
+	// nodes powered by farmerbot report every 24 hours, if didn't report for 1 cycle, it's marked down
+	nodeUpStateFactor         = 2
```
These variables are repeated throughout the code, perhaps they should be moved to their own module, and all shared logic can be moved there as well, to make sure that it is consistent across the code base

alichaddad commented on grid-proxy/internal/explorer/converters.go:
```diff
@@ -11,16 +11,31 @@ import (
+	// nodes powered by farmerbot report every 24 hours, if didn't report for 1 cycle, it's marked down
+	nodeUpStateFactor         = 2
+	nodeUpReportInterval      = time.Minute * 40
+	nodeStandbyStateFactor    = 1
```
Please add docs for this constant

Omarabdul3ziz committed:
assemble nodestatus code in single package

- move the duplicated code to nodestatus
- add docs in an md file instead of comments

alichaddad reviewed (approved):
LGTM
